zone/client_packet.cpp

```
@@ -11016,18 +11332,46 @@ void Client::Handle_OP_MoveMultipleItems(const EQApplicationPacket *app)
 
 				if (move.item) {
 					items.push_back(move);
-					database.SaveInventory(CharacterID(), NULL, from_slot); // We have to manually save inventory here.
+					if (!database.SaveInventory(CharacterID(), NULL, from_slot)) {
+						operation_valid = false;
+						break;
+					}
 				} else {
```

Clang (14.0.6)

[note] +1, nesting level increased to 4

(clang)